common/api/imodeljs-frontend.api.md

```diff
@@ -8307,7 +8325,7 @@ export class ScreenViewport extends Viewport {
     openToolTip(message: HTMLElement | string, location?: XAndY, options?: ToolTipOptions): void;
     readonly parentDiv: HTMLDivElement;
     // @internal (undocumented)
-    pickCanvasDecoration(pt: XAndY): CanvasDecoration | undefined;

```

Comment:
Is this import here weird?

Reply:
extract-api is weird. I don't worry about it unless the PR build complains.